src/main/java/com/splunk/logging/HttpEventCollectorSender.java

```diff
@@ -155,7 +173,16 @@ public HttpEventCollectorSender(
     }
 
   }
-  
```

Comment:
my preference is for the instruction to startPollingForHealth not come from the Sender at all. I'd prefer to see startPollingForHealth in AckManager.